src/models/device.ts

```diff
@@ -342,6 +351,7 @@ const getDeviceModel = function (
 	async function startOsUpdate(
```

Comment:
I believe the alternate signatures for this function above also need an options parameter.